fuzz/quic-srtm.c

```diff
@@ -60,7 +61,7 @@ int FuzzerTestOneInput(const uint8_t *buf, size_t len)
         if (!PACKET_get_1(&pkt, &cmd))
             goto err;
 
-        switch (cmd) {
+        switch (cmd % CMD_MAX) {
```

Comment:
I agree with @kroeckx and @mattcaswell, this change (modulo) operation should be combined with stuff found in [#24831](https://github.com/openssl/openssl/pull/24831)